dask/diagnostics/profile_visualize.py

```diff
@@ -13,7 +13,7 @@
 from ..core import istask
 
 
-def pprint_task(task, keys):
+def pprint_task(task, keys, budget=60):
```

Comment:
This kwarg should be added to the docstring. It also might read easier as `max_task_length`, or `label_size` or something a bit more descriptive. Not a major gripe though.